翻译客服文档 (#833)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/tap-support/features/setup.mdx

---
title: Open TapSupport Privilege
sidebar_label: Opening process
sidebar_position: 1
---

> To use Tap Customer Service, you need to open it in the TapTap Developer Center.

Reviewer: ```suggestion
To use TapSupport Service, you need to activate it in the TapTap Developer Center.
```

### Enable Service

> Game Customer Service is a vendor dimension service, developers can self-enable the service at **Developer Center Vendor Page > Game Service > Game Customer Service**.

Reviewer: ```suggestion
TapSupport Service is a vendor dimension service, developers can self-enable the service at **Developer Center Home Page > Game Service > TapSupport Service**.
```

:::info Permission Requirements

> Access to the Game Customer Service module in the Developer Center requires the **Customer Service Administrator** permission in the vendor permissions.

Reviewer: ```suggestion
Access to the TapSupport Service module in the Developer Center requires the **Customer Service Administrator** permission in the vendor permissions.
```

:::

### Price program

> The customer service system is charged according to the number of activated customers.

Reviewer: ```suggestion
The TapSupport system is charged according to the number of activated customers.
```

:::info
There is no billing during the beta period, we will post a notice in advance before the official billing.
:::

> ### Add Customer Service

Reviewer: ```suggestion
### Add Customer Service Staff
```

> After enabling customer service, you first need to add customer service.

Reviewer: ```suggestion
After enabling customer service, you first need to add customer staff.
```

> You can switch to **Customer Service Management** tab to add customer service. To add customer service, you need to provide the cell phone number of the customer service, and the customer service will receive an invitation SMS after you add the customer service. Customer service through the cell phone plus SMS verification code to log in the customer service background.

Reviewer: ```suggestion
You can switch to **Customer Service Management** tab to add customer service. To add customer service, you need to provide the mobile phone number of the customer service, and the customer service will receive an invitation SMS after you add the customer service. Customer service logs in to the customer service backend by adding the SMS verification code via mobile phone.
```

The SMS received by the customer service will have a link to the customer service workbench, you can also find the customer service workbench in the upper right corner of the game customer service module in the developer center to jump to the entrance.

> When adding customer service, you need to specify the role and scope of the customer service, the responsibilities and specific permissions of different roles see ["Permission Control of Customer Service"](/sdk/tap-support/features/roles/).

Reviewer: ```suggestion
When adding customer service, you need to specify the role and scope of the customer service, the responsibilities and specific authorities of different roles see ["Authority Control of Customer Service"](/sdk/tap-support/features/roles/).
```

:::info Be the first customer service!

> Vendor administrators can only perform account and security related configurations in the DC backend, specific customer service business related configurations need to be performed in the Customer Service Workbench. To configure the service, you can add yourself as an administrator.

Reviewer: ```suggestion
Super administrators can only perform account and security related configurations in the DC backend, specific customer service business-related configurations need to be performed in the Customer Service Workbench. To configure the service, you can add yourself as an administrator.
```
